tests/operator/operator.go

```diff
@@ -3389,3 +3375,16 @@ func patchKV(name string, data []byte) {
 		return err
 	}).WithTimeout(10 * time.Second).WithPolling(1 * time.Second).Should(Succeed())
 }
+
+func parseDaemonset(name string) string {
```

Comment:
nit (out-of-scope as this name is not presented by this PR), but should we take the opportunity to improve this name? Maybe `getDaemonsetImage`?